rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -0,0 +1,254 @@
+
+servers:
+  - url: ${rest.server.url}
+
+paths:
+  /mis/product:
+    post:
+      summary: Create a Product
+      description: >
+        Clients send a file attachment to create a Product resource.
+      operationId: storeProduct
+      tags:
+        - product
+      parameters:
+        - $ref: '#/components/parameters/Accept-Version'
+      requestBody:
+        required: true
+        description: >
+          The body of the ingest request includes an attached file.
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                file:
+                  description: >
+                    The actual file attachment that is sent in the request. The current maximum is the character
+                    equivalent of 10GB.
+                  type: string
+                  format: binary
+                  minLength: 1
+                  maxLength: 14316557655
+              required:
+                - file
+      responses:
+        '201':
+          $ref: '#/components/responses/Created'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '501':
+          $ref: '#/components/responses/NotImplemented'
+        'default':
+          $ref: '#/components/responses/DefaultError'
+  /mis/product/{productID}:
+    get:
+      summary: Get a Product
+      description: Client sends a Product ID to retrieve the Product as a file.
+      operationId: retrieveProduct
+      tags:
+        - product
+      parameters:
+        - $ref: '#/components/parameters/Accept-Version'
+        - name: productID
+          in: path
+          required: true
+          schema:
+            type: string
+            example: "123e4567e89b12d3a456426655440000"
+      responses:
+        200:
+          description: Get Product
+          content:
+            application/octet-stream:
+              schema:
+                type: string
+                format: binary
+          headers:
+            'Content-Version':
+              $ref: '#/components/headers/ContentVersion'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '501':
+          $ref: '#/components/responses/NotImplemented'
+        'default':
+          $ref: '#/components/responses/DefaultError'
+
+components:
+  responses:
+    OK:
+      description: >
+        The service has fulfilled the request.
+      headers:
+        'Content-Version':
+          $ref: '#/components/headers/ContentVersion'
+    Accepted:
+      description: >
+        The request was accepted.
+      headers:
+        'Content-Version':
+          $ref: '#/components/headers/ContentVersion'
+    Created:
```

Comment:
✏️ Inline `/components/responses/Created` because it is unique for create request